setup.md

```diff
@@ -1,7 +1,25 @@
 ---
 title: Setup
 ---
-FIXME
 
+This tutorial requires a "shell" or "terminal" program. Below are instructions for installing this program on different operating systems. For more information on how to use the terminal program, see [link](link url).
+
+### Windows Installation and Access
+
+If you are using Windows, you will need to install [the Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install). WSL will have the ssh programs you need by default, so once you've installed WSL you are ready for this tutorial.
```

Comment:
Note that newer Windows systems have SSH pre-installed in the Windows shell and you can use it in the same way. However, I'm not sure how it behaves regarding configs and the like. But figuring that out and perhaps adding a note could be something to figure out.

Reply:
Noted. Will look into and add whatever is relevant.